frontend/pages/smileLoginRoute/components/MemberPanel/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import styles from './style';
+import getConfig from '../../../../helpers/getConfig';
+
+const { shopifyAlias } = getConfig();
+
+/**
+ * Member component Smile Login Panel
+ * @param {Object} memberText Object containing header info
+ * @return {JSX}
+ */
+const MemberPanel = ({ memberText }) => {
+  if (!memberText) {
+    return null;
+  }
+  const { header, paragraph } = memberText || {};
```

Comment:
memberText can't never be falsy here because of your return above

Reply:
Removed falsy check